tools/SetupFlow/DevHome.SetupFlow/Models/Environments/CreateEnvironmentTask.cs

```diff
@@ -1,198 +1,198 @@
-            // event. Since the call flow is disjointed an extension may not have sent the session ended event when this method is called.
-            _autoResetEventToStartCreationOperation.WaitOne(TimeSpan.FromMinutes(1));
-
-            if (string.IsNullOrWhiteSpace(UserJsonInput))
-            {
-                // The extension's creation adaptive card may not need user input. In that case, the user input will be null or empty.
-                _log.Information("UserJsonInput is null or empty.");
-            }
-
-            // If the provider details are null, then we can't proceed with the operation. This happens if the auto event times out.
-            if (ProviderDetails == null)
-            {
-                _log.Error("ProviderDetails is null so we cannot proceed with executing the task");
-                AddMessage(_stringResource.GetLocalized(StringResourceKey.EnvironmentCreationFailedToGetProviderInformation), MessageSeverityKind.Error);
-                return TaskFinishedState.Failure;
-            }
-
-            var sdkCreateEnvironmentOperation = ProviderDetails.ComputeSystemProvider.CreateCreateComputeSystemOperation(DeveloperIdWrapper.DeveloperId, UserJsonInput);
-            var createComputeSystemOperationWrapper = new CreateComputeSystemOperation(sdkCreateEnvironmentOperation, ProviderDetails, UserJsonInput);
-
-            // Start the operation, which returns immediately and runs in the background.
-            createComputeSystemOperationWrapper.StartOperation();
-            AddMessage(_stringResource.GetLocalized(StringResourceKey.EnvironmentCreationForProviderStarted), MessageSeverityKind.Info);
-
-            _computeSystemManager.AddRunningOperationForCreation(createComputeSystemOperationWrapper);
-            CreationOperationStarted = true;
-
-            _log.Information("Successfully started the creation operation");
-            return TaskFinishedState.Success;
-        }).AsAsyncOperation();
-    }
-
-    IAsyncOperation<TaskFinishedState> ISetupTask.ExecuteAsAdmin(IElevatedComponentOperation elevatedComponentOperation)
-    {
-        return Task.Run(() =>
-        {
-            // No admin rights required for this task. This shouldn't ever be invoked since the RequiresAdmin property is always false.
-            _log.Error("Admin execution is not required for the create environment task");
-            return TaskFinishedState.Failure;
-        }).AsAsyncOperation();
-    }
-
-    private void Dispose(bool disposing)
-    {
-        if (!_disposedValue)
-        {
-            if (disposing)
-            {
-                _autoResetEventToStartCreationOperation.Dispose();
-            }
-
-            _disposedValue = true;
-        }
-    }
-
-    public void Dispose()
-    {
-        // Do not change this code. Put cleanup code in 'Dispose(bool disposing)' method
-        Dispose(disposing: true);
-        GC.SuppressFinalize(this);
-    }
-}
+﻿// Copyright (c) Microsoft Corporation.

```

Comment:
is this a new line change?

Reply:
This one had a lot of LF->CR/LF changes. I can't figure out how we get those LF line endings. I didn't change anything in this file, but VS corrected line endings, so I kept it. Really annoying, but after almost every merge with main I get those LFs.